src/main/java/org/jenkinsci/test/acceptance/docker/fixtures/GitContainer.java

```diff
@@ -44,4 +45,14 @@ public String getRepoUrlInsideDocker(String alias) throws IOException {
         return "ssh://git@" + alias + REPO_DIR;
     }
 
+    /**
+     * Add an additional certificate to <code>~/.ssh/authorized_keys</code>
+     * @param cert the certificate public key
+     */
+    public void addSSHCertificate(String cert) throws IOException, InterruptedException {
+        Docker.cmd("exec", getCid()).add("/bin/bash",  "-c",  "echo " + cert + " >> /home/git/.ssh/authorized_keys")
```

Comment:
NIT: would not this be the result of the String builder on exec, id, /bin/bash, -c etc etc?
Just wondering why you passed some in the initializer and some others in the .add

Reply:
Yeah, it would be the same I think, but I wanted to separate them on different calls for clarity.